moveit_core/collision_detection_bullet/CMakeLists.txt

```
@@ -48,8 +48,7 @@ target_link_libraries(collision_detector_bullet_plugin
 
 install(DIRECTORY include/ DESTINATION include)
 install(FILES ${CMAKE_CURRENT_BINARY_DIR}/${MOVEIT_LIB_NAME}_export.h DESTINATION include)
-install(TARGETS ${MOVEIT_LIB_NAME} EXPORT ${MOVEIT_LIB_NAME}
-  TARGETS collision_detector_bullet_plugin EXPORT collision_detector_bullet_plugin
+install(TARGETS ${MOVEIT_LIB_NAME} collision_detector_bullet_plugin EXPORT ${MOVEIT_LIB_NAME}
```

Shouldn't this be something like this?
```suggestion
install(TARGETS ${MOVEIT_LIB_NAME} collision_detector_bullet_plugin EXPORT export_${MOVEIT_LIB_NAME}
ament_export_targets(export_${MOVEIT_LIB_NAME} HAS_LIBRARY_TARGET)
```

---

Both should work

---

I think it'd be nice to stick to one style. I almost asked the same thing before seeing this conversation, so it could cause more confusion in the future